x-pack/plugins/security_solution/common/detection_engine/utils.ts

```diff
@@ -68,3 +69,16 @@ export const normalizeThresholdObject = (threshold: Threshold): ThresholdNormali
 
 export const normalizeMachineLearningJobIds = (value: string | string[]): string[] =>
   Array.isArray(value) ? value : [value];
+
+export const isSuppressibleAlertRule = (ruleType: Type): boolean => {
+  const suppressibleRules: string[] = Object.values(SuppressibleAlertRules);
+  return suppressibleRules.includes(ruleType);
```

Comment:
do we need to convert `SuppressibleAlertRules` to array?

```suggestion
  return ruleType in SuppressibleAlertRules;
```

Won't just checking property in object work?

Reply:
The `in` operator is primarily designed to check for the existence of properties in objects, and its behavior with enums can be less straightforward.

For enums in TypeScript, the values are typically reverse-mapped to their keys, and checking for a specific value using it might not behave as expected.

So for a reliable check, it is better to go with `Object.values` and `includes` when working with TS enums